views/auth/updateaccount.handlebars

```diff
@@ -0,0 +1,322 @@
+<!-- GET: form with all account fields except password, prefilled with current information in database -->
+{{#if data}}
+{{#each data}}
+<!-- form renders here, prefilled with data from user -->
+<div class="col-xl-8 col-lg-8 col-md-10 col-sm-12 col-12">
+    <h3 class="pb-3">{{title}}</h3>
+    <div class="pb-3">
+        <div>Looking to update your password? <a href="/resetpassword">Click here</a></div>
+    </div>
+
+    <form action='/updateaccount' method="POST">
+        <div class="form-row">
+            <div class="form-group col-6">
+                <label for="firstName">First Name</label>
+                <input type="text" class="form-control" name="firstName" value="{{firstName}}" placeholder="First name"
+                    required>
+            </div>
+            <div class="form-group col">
+                <label for="lastName">Last Name</label>
+                <input type="text" class="form-control" name="lastName" value="{{lastName}}" placeholder="Last name"
+                    required>
+            </div>
+        </div>
+        <div class="form-row">
+            <div class="form-group col-12">
+                <label for="Email">Email</label>
+                <input type="email" class="form-control" name="email" value="{{email}}" placeholder="Email" required>
+            </div>
+        </div>
+        <div class="form-row">
+            <div class="form-group col-12">
+                <label for="Address">Address</label>
+                <input type="text" class="form-control" name="address" value="{{address}}"
+                    placeholder="1343 Anywhere St Apt 100" required>
+            </div>
+        </div>
+        <div class="form-row">
+            <div class="form-group col-6">
+                <label for="city"> City</label>
+                <input type="text" class="form-control" name="city" value="{{city}}" required>
+            </div>
+            <div class="form-group col">
+                <label for="state">State</label>
+                <input type="text" class="form-control" name="state" value="{{state}}" required>
+            </div>
+            <div class="col">
+                <label for="postalCode">Zip</label>
+                <input type="text" class="form-control" name="postalCode" value="{{postalCode}}" required>
+            </div>
+        </div>
+        <div class="form-row">
+            <div class="form-group col-12">
+                <label for="country">Country</label>
+                <select class="form-control" name="country" required>
+                    <option value="" selected="selected">Select a Country</option>
+                    <option value="AF">Afghanistan</option>
```

Comment:
As discussed, now that this is used on two pages lets try to create this list in a function server side and pass it in to the view with the data that way it'll clean up the handlebars and make it more re-usable. You don't have to do it for this time, it could be one of your code smells/refactors for this sprint!